core/src/block_sync.rs

```diff
@@ -94,8 +96,7 @@ pub struct ContinueSync;
 /// Message for getting updates from other peers
 ///
 /// Starts the `BlockSync`, meaning that every `gossip_period`
-/// the peers would gossip about latest block hashes
-/// and try to synchronize their blocks.
+/// peer would poll randomly selected `sync_peers_count` peers for their blocks
```

Comment:
```suggestion
/// peer would poll randomly selected `sync_peers_count` peers for their latest block hashes
/// and try to sync in case it has a different block hash.
```